.github/workflows/ci.yml

```diff
@@ -74,10 +74,10 @@ jobs:
         include:
         ## HIP
         - name: linux_hip_nvcc-9.2_gcc-5_debug_hip_only
-          os: ubuntu-latest
+          os: rocm/pytorch:rocm3.5_ubuntu16.04_py3.6_pytorch
```

Comment:
@BenjaminW3 I expect this is the place to load a different base container or? I will switch to a container with clean installed hip-clang.

Reply:
I do not think that this works. The `os:` only allows one of the few virtual machine types which github allows.

Reply:
`ALPAKA_CI_DOCKER_BASE_IMAGE_NAME` is the correct one

Reply:
thx